src/react-apps/applications/ux-editor/src/components/config/EditModalContent.tsx

```diff
@@ -408,12 +421,54 @@ class EditModalContentComponent extends React.Component<IEditModalContentProps,
         );
       }
 
+      case 'TextArea': {
+        const component = (this.state.component as IFormTextAreaComponent);
+        return (
+          <Grid item={true} xs={12}>
+            {renderSelectDataModelBinding(
+              this.props.component.dataModelBindings,
+              this.handleDataModelChange,
+              this.props.language)}
+            {renderSelectTextFromResources('modal_properties_label_helper',
+              this.handleTitleChange,
+              this.props.textResources,
+              this.props.language,
+              this.props.component.textResourceBindings.title)}
+            {renderSelectTextFromResources('modal_properties_description_helper',
+              this.handleDescriptionChange,
+              this.props.textResources,
+              this.props.language,
+              this.props.component.textResourceBindings.description)}
+            <Grid item={true} classes={{ item: this.props.classes.gridItem }}>
+              {getLanguageFromKey('ux_editor.modal_properties_read_only_description', this.props.language)}
+            </Grid>
+            <Grid item={true}>
+              <Checkbox
+                classes={{ root: this.props.classes.checkbox }}
+                checked={!!component.readOnly}
+                onChange={this.handleReadOnlyChange}
+              />
```

Comment:
Check if there is a shared component, or create one for reuse.

Reply:
Created a shared AltinnCheckBox.

Reply:
Agreed, fixed.